controllers/mod/artefacts_data.rb

```diff
@@ -1,66 +1,96 @@
 class ArtefactsdataController < ApplicationController
-  namespace "/artefacts/:artefactID/resources" do
-    
-    get do
-      ontology, latest_submission = get_ontology_and_submission(ontology_acronym: params["artefactID"])
-      check_access(ontology)
-      _, page, size = settings_params(LinkedData::Models::Class).first(3)
-      size_per_type = [size / 6, 1].max
+  namespace "/mod-api" do
+    namespace "/artefacts/:artefactID/resources" do
 
-      types = [
-        LinkedData::Models::Class,
-        LinkedData::Models::Instance,
-        LinkedData::Models::SKOS::Scheme,
-        LinkedData::Models::SKOS::Collection,
-        LinkedData::Models::SKOS::Label
-      ]
+      doc('Artefact', "Get a list of all the resources within an artefact") do
+        path_parameter('artefactID', type: 'string', description: 'The acronym of the artefact', default: "STY")
```

Comment:
this 4 lines are reptead a lot, extract it in a helper, it will make the documentation easier